core-tests/src/test/java/org/mule/message/processing/AsyncResponseFlowProcessingPhaseTestCase.java

```diff
@@ -138,7 +138,7 @@ public void order()
     @Test
     public void runPhaseWithMessagingExceptionThrown() throws Exception
     {
-        when(mockContext.supportsAsynchronousProcessing()).thenReturn(true);
```

Comment:
the default value for supportsAsynchronousProcessing() was changed in  AsyncResponseFlowProcessingPhase. The same change was missing in this test case